opal/util/convert-help-files-to-c-code.py

```diff
@@ -17,9 +17,25 @@ def find_help_files(root, verbose=False):
     # some directories (e.g., 3rd-party)
     help_files = []
     skip_dirs = ['.git', '3rd-party']
+    include_top_dirs = ['ompi', 'opal', 'oshmem']
```

Comment:
It would be nicer if the list of directories to scan comes instead from the command line argument, such that make can pass `$(MCA_PROJECT_SUBDIRS)` into.

Reply:
Done.